charts/clusterapi-resources/templates/KubeadmConfigTemplate.yaml

```diff
@@ -11,9 +11,12 @@ spec:
     spec:
       joinConfiguration:
         nodeRegistration:
-          criSocket: /var/run/containerd/containerd.sock
+          criSocket: unix:///var/run/containerd/containerd.sock
```

Comment:
These changes will cause a rollout on control-plane nodes when cluster Applications on Argo pick the new chart version.

That rollout will be needed for pacman's #213 anyway, and I have an idea on how to orchestrate that so only one rollout will be needed.